workspaces/applications/crates

```diff
@@ -0,0 +1 @@
+../..
```

Comment:
Question (not needing a change): What's this for?

Reply:
Workspaces come with an expectation that all source would be in sub-directories. In order to share code between workspaces and keep working with the current code organization, I create a `crates` symbolic link that points to the Veracruz top-level directory.

Note: eventually we should move all crates from Veracruz top-level directory into a `crates` director, the `crates` symbolic links in the workspaces will be updated, and that should help reduce the confusion.